simple_history/management/commands/clean_duplicate_history.py

```diff
@@ -69,7 +69,7 @@ def _process(self, to_process, date_back=None, dry_run=True):
             # have changes (in the given period) but
             # `m_qs.values(model._meta.pk.name).distinct()`
             # is actually slower than looping all and filtering in the code...
-            for o in model.objects.all():
+            for o in model.objects.all().iterator():
```

Comment:
do you need `.all()` or can you just use `.iterator()`? Also, was just looking at the docs. Will this make a separate db query for every for loop? Or am I reading that wrong?

Reply:
My understanding is that the iterator will get 2,000 records at a time and keep a cursor open on the database.

It appears the .all().iterator() can be trimmed to .iterator(). I'll make that change.